Makefile

```diff
@@ -110,6 +111,12 @@ fmt: install-tools ## Formats all go files
 	$(GO) run .github/tools/matrixchecker/main.go
 	./build/docker-go-version.sh Dockerfile ./suppression-backup-service/Dockerfile
 
+.PHONY: sec
+sec: ## Run security checks
+	$(GO) run $(gitleaks) detect .
+	$(GO) run $(govulncheck) ./...
+	trufflehog filesystem . --no-verification -x .trufflehogignore
```

Comment:
Do we need to install something for `trufflehog`?

Reply:
I've removed trufflehog in favour of just gitleaks for now